hld/Api.hx

```diff
@@ -1,6 +1,6 @@
 package hld;
 
-@:enum abstract WaitResult(Int) {
+@:enum abstract WaitResult(Int) from Int to Int {
```

Comment:
I don't like much from Int/to Int  as this breaks the whole idea of using an abstract. You can either use cast in your implementation or add static function fromInt + function toInt() return this so the conversion is explicit. Same for Register.

Reply:
Will do. I use to/from all the time in my own code though 🤨